.github/workflows/test.yml

```diff
@@ -380,7 +382,7 @@ jobs:
       matrix:
         python-version: [ "3.11" ]
         airflow-version: [ "2.11", "3.0" ]
-        dbt-version: ["1.5", "1.6", "1.7", "1.8", "1.9", "1.10"]
+        dbt-version: ["1.6", "1.7", "1.8", "1.9", "1.10"]
```

Comment:
Why are you removing the dbt-1.5? Do we need to update some docs, too?

Reply:
We still have other tests running against dbt 1.5 - I just removed this one to simplify the already complex usage of `altered_jaffle_shop` and `jaffle_shop` depending on the dbt version.